code/modules/mining/mine_turfs.dm

```diff
@@ -1,20 +1,18 @@
 /**********************Mineral deposits**************************/
 
-var/global/list/rockTurfEdgeCache
-#define NORTH_EDGING	"north"
-#define SOUTH_EDGING	"south"
-#define EAST_EDGING		"east"
-#define WEST_EDGING		"west"
-
 /turf/simulated/mineral //wall piece
 	name = "rock"
-	icon = 'icons/turf/mining.dmi'
-	icon_state = "rock_nochance"
+	icon = 'icons/turf/smoothrocks.dmi'
+	icon_state = "rock"
+	smooth = SMOOTH_MORE
+	canSmoothWith = list (/turf/simulated/mineral, /turf/simulated/wall)
 	baseturf = /turf/simulated/floor/plating/asteroid/airless
 	oxygen = 0
 	nitrogen = 0
 	opacity = 1
 	density = 1
+	pixel_y = -4
+	pixel_x = -4
 	blocks_air = 1
 	layer = TURF_LAYER + 0.05
```

Comment:
Yeah, if we ever stop the teleport proc from throwing people into dense turfs, we can make it a higher layer.